app/packs/src/apps/mydb/elements/details/VersionsTable.js

```diff
@@ -0,0 +1,208 @@
+      }
+      case 'wellplate': {
+        WellplatesFetcher.fetchById(id).then((result) => {
+          parent.setState({ wellplate: result });
+        });
+        break;
+      }
+      default:
+        // do nothing
+    }
+
+    elementShowOrNew({
+      type: entityType,
+      params: { [`${entityType}ID`]: id }
+    });
+  };
+
+  handleRevert = (changes) => VersionsFetcher.revert(changes)
+    .then(() => this.fetchVersions())
+    .then(() => this.reloadEntity());
+
+  fetchVersions() {
+    const { type, id } = this.props;
+    const { page } = this.state;
+
+    return VersionsFetcher.fetch({
+      type, id, page
+    }).then((result) => {
+      if (!result) return false;
+
+      return this.setState({
+        versions: result.elements || [],
+        page: result.page || 1,
+        pages: result.pages || 1,
+      });
+    });
+  }
+
+  render() {
+    const { versions, page, pages } = this.state;
+    const { isEdited } = this.props;
+
+    const pagination = () => (
+      <Pager>
+        <Pager.Item
+          previous
+          href="#"
+          onClick={() => this.handlePagerClick('prev')}
+          disabled={page >= pages}
+        >
+          &larr; Older Versions
+        </Pager.Item>
+        <Pager.Item
+          next
+          href="#"
+          onClick={() => this.handlePagerClick('next')}
+          disabled={page <= 1}
+        >
+          Newer Versions &rarr;
+        </Pager.Item>
+      </Pager>
+    );
+
+    const columns = [
+      {
+        dataField: 'caret',
+        text: '',
+        isDummyField: true,
+        // eslint-disable-next-line react/no-unstable-nested-components
+        formatter: () => <i className="fa fa-caret-right history-table__caret" />,
+      },
+      {
+        dataField: 'id',
+        text: '#',
+      },
+      {
+        dataField: 'createdAt',
+        text: 'Modified on',
+        // eslint-disable-next-line react/no-unstable-nested-components
+        formatter: (cell) => (
+          <VersionsTableTime dateTime={cell} />
+        ),
+      },
+      {
+        dataField: 'userName',
+        text: 'Author',
+      },
+    ];
+
+    const expandRow = {
+      onlyOneExpanding: true,
+      parentClassName: 'active',
+      renderer: (row) => (
+        <VersionsTableChanges
+          id={row.id}
+          changes={row.changes}
+          handleRevert={this.handleRevert}
+          isEdited={isEdited}
+        />
+      ),
+    };
+
+    return (
+      <>
+        <style>
+          {`
+            .reset-expansion-style {
+              background: white;
+            }
+          `}
+        </style>
+        <ul className="history-legend">
+          <li className="history-legend__item history-legend__item--old">before</li>
+          <li className="history-legend__item history-legend__item--new">after</li>
+          <li className="history-legend__item history-legend__item--current">current value</li>
+        </ul>
+        <BootstrapTable
```

Comment:
Maybe use `ag-grid` instead? We already have it in our dependencies.  Even better would be to use a plain `Table` from `react-bootstrap`.